### ISSUE TITLE
Where would you like to create your workspace - question missing from Installation guide

### ISSUE THREAD

**Issue by BinaryGeometry** [spartan-ng/spartan#321]
### Which scope/s are relevant/related to the feature request?

Documentation

### Information

There is a step in the CLI between "You will be asked to choose a folder name for your workspace:" and "Pick whichever fits your project the best!" that is not covered in the Installation docs.

✔ Where would you like to create your workspace? · mars
? Set up CI with caching, distribution and test deflaking …  
(it's free and can be disabled any time)
Yes, for GitHub Actions with Nx Cloud
Yes, for CircleCI with Nx Cloud
Skip for now

As a weary user I hate these kind of questions, especially when there is no guidance related to them. I want to get going fast and suddenly I'm off on a research journey again (after thinking I'd finally arrived). 

Just a little guidance here would be so reassuring. Can I enable at any time "don't worry changing your mind is just a yaml entry away" or is it a whole different paradigm I'm entering into such as Angulars ? Use Standalone (recommended).

### Describe any alternatives/workarounds you're currently using

Prayer

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ x ] No - simply because it's deep knowledge I don't have.

**Comment by BinaryGeometry** [comment]
Realise this is an nx thing and not related

**Comment by BinaryGeometry** [comment]
Also realised that "Where would you like to create your workspace?" is an nx thing as well, so maybe you would like to add a note to your documentation, save someone else a journey to the 404 at the end of the the [https://nx.dev/getting-started/intro]. But yeah, do feel free to close.

**Comment by goetzrobin** [comment]
Yes these are Nx specific questions. Feel free to add a note to the documentation and I am happy to merge a PR

**Comment by ashley-hunter** [comment]
Closing as this is Nx related rather than Spartan related.
